operations/charge.go

```diff
@@ -119,13 +124,27 @@ func (req *CreateCharge) Op() *internal.Op {
 type UpdateCharge struct {
 	ChargeID    string `query:"-"`
 	Description string
+	Metadata    map[string]interface{}
+}
+
+func (req *UpdateCharge) MarshalJSON() ([]byte, error) {
```

Comment:
@may I think what we need is `omitempty` krub

```
type UpdateCharge struct {
  Description string `json:",omitempty"`
  Metadata map[string]interface{} `json:",omitempty"`
}
```

Reply:
@Kanokorn Other operations can use `omitempty` krub except `CreateCharge` because field `DontCapture` because capture params will set `false` if this field is true krub. But I think I have the way to manage this better.